lib/membrane_webm/serializer/webm.ex

```diff
@@ -106,6 +106,7 @@ defmodule Membrane.WebM.Serializer.WebM do
        # 2 for audio
        TrackType: 2,
        CodecID: "A_OPUS",
+       FlagLacing: 0,
```

Comment:
As we everywhere put 0 as value for `FlagLacing`, I would add 0 as constant with a name something like `no_lacing`.